.eleventy.js

```diff
@@ -1,20 +1,25 @@
 const htmlmin = require('html-minifier')
-const articleSections = ['html', 'css', 'js', 'tools']
-const articleFormats = ['articles', 'doka']
 
 module.exports = function(config) {
+  const articleSections = ['html', 'css', 'js', 'tools']
+
   // Add all Tags
-  articleSections.forEach((el) => {
-    config.addCollection(el, function(collectionApi) {
-      return collectionApi.getFilteredByGlob(`src/${el}/**`);
-    });
-  });
+  articleSections.forEach((element) => {
+    let elementArticles = element + 'Articles'
+    let elementDoka = element + 'Doka'
+
+    config.addCollection(element, function(collectionApi) {
+      return collectionApi.getFilteredByGlob(`src/${element}/**`)
```

Comment:
Кажется так будет точнее для всех случаев. Заодно проигнорируем «в работе»
```suggestion
      return collectionApi.getFilteredByGlob(`src/${element}/**/index.md`) 
```

Reply:
Если так сделать, надо будет поправить подключение списков немного. Попробую.

Reply:
Поправлено в [d456c82](https://github.com/Y-Doka/platform/pull/45/commits/d456c82425de6e58ab36d8d45ec956fb4eb398a5)

Reply:
Стало как надо! я вчера экспериментировал и тоже столкнулся с ситуацией, когда в коллекции появлялась куча мусора